taxonomies/labels.txt

```diff
@@ -23113,3 +23113,65 @@ description:fr: En choisissant des produits Prix Juste Producteur, on vous garan
 image:fr: prix-juste-producteur.90x90.png
 web:fr: https://prixjuste.be/
 
+########################## Canadian front-of-package nutrition symbol ##########################
+
+# There are 56 possible variations. 7 different combinations for the ingredients, 4 different language versions and 2 orientations (vertical/horizontal).
+
+en: Canadian Front-of-package nutrition symbol
+fr: Symbole nutritionnel sur le devant de l'emballage
+country:en: en:Canada
+description:en: We're improving food labels. A front-of-package nutrition symbol is required on foods that are high in one or more of these nutrients: sodium, sugars, saturated fat. The food industry has been given until January 1, 2026 to make this change. However, you may start seeing the front-of-package nutrition symbol earlier.
+label_categories:en: nutritional-values
+web:en: https://www.canada.ca/en/health-canada/services/food-nutrition/nutrition-labelling/front-package.html
+web:fr: https://www.canada.ca/fr/sante-canada/services/aliments-nutrition/etiquetage-nutritionnel/devant-emballage.html
+
+< en:Canadian front-of-package nutrition symbol
+en: High in Sat fat Sugars Sodium (Health Canada) - Horizontal
```

Comment:
```suggestion
en: High in Sat fat, Sugars and Sodium (Health Canada)
```

Reply:
This is what the user will read on the product page. There is no need to add "horizontal", it could be confusing for him/her. And I have added punctuation and a final "and" to make it better to read.